test/Escrow.test.js

```diff
@@ -80,7 +80,7 @@ contract('Escrow', (accounts) => {
 
     it('fails when aborting with different address', async () => {
       try {
-        tx = await Escrow.abort({ from: externalAddress });
```

Comment:
this doesn't look correct, tx is used on the next line. 
and I think this test won't break if someone were to remove the "trusted" modifier on the "abort" method in the escrow contract